docs/articles/user_acquisition_analytics.md

```diff
@@ -0,0 +1,441 @@
+
+To understand which ad creatives generated how many signups, we create a DataFrame:
+
+```python
+pd.DataFrame(user_df["ad_creative"].value_counts())
+```
+
+...which looks like this:
+
+![ad_creatives by count](../assets/use_cases/user_acquisition_analytics/ad_creative-count.png)
+
+The influencer (XYZCr$$d) backed ad creatives seem to have worked better - generating many more signups (total 5145) than the August campaign ad creatives (total 2855). 
+
+Now, let's take a look at the **distribution of users by activity level** (api calls/day).
+
+```python
+alt.Chart(user_df).mark_bar().encode(
+    alt.X("activity:Q", bin=True, title="Activity count"),
+    alt.Y("count()", title="# of users"),
+).properties(width=600, height=400)
+```
+
+![distribution by activity count](../assets/use_cases/user_acquisition_analytics/user_distrib-activity_count.png)
+
+The activity (api calls/day) distribution is bimodal. Here, it's possible that the first activity level group (0.00-0.40 api calls/day) may be largely new users, who - if they signed up in the most recent (December) campaign - have less time to accumulate activity than users who signed up earlier. (Note: we can use derive our NumberSpace min and max from this distribution.)
+
+Now let's examine the distribution of new users per signup date.
+
+```python
+dates: pd.DataFrame = pd.DataFrame(
+    {"date": [str(datetime.fromtimestamp(ts).date()) for ts in user_df["signup_date"]]}
+)
+dates_to_plot = pd.DataFrame(dates.value_counts(), columns=["count"]).reset_index()
+alt.Chart(dates_to_plot).mark_bar().encode(
+    alt.X("date", title="Signup date"), alt.Y("count", title="Number of subscribers")
+).properties(height=400, width=1200)
+```
+
+![new users per signup date](../assets/use_cases/user_acquisition_analytics/new_users-signup_date.png)
+
+This distribution confirms that our second campaign (December) works much better than the first (August). The jump in signups at 2023-12-21 coincides with our second campaign, and our data is exclusively users who signed up by clicking on campaign ad_creatives. To get more insights from our user acquisition analysis, we need two periods that fit our distribution. A first period of 65 days and a second of 185 days seem appropriate.
+
+Of our 8k users, roughly 2k subscribed in the first campaign period (65 days), and 6k in the second. Since we already know which ad_creatives users clicked on to go sign up (2855 on old campaign ads, 5145 on new campaign ads), we know that some of the roughly 6k users who signed up after the second (December) campaign started clicked on old (August) campaign ads *after* the new campaign ad began (possibly after seeing the new campaign ads).
+
+So far, we know that:
+
+- many more users signed up in response to the influencer-backed ad creatives (second campaign) than the first campaign
+- which specific ad creatives generated the most user signups
+- the vast majority of users have low to moderate activity
+
+We don't know:
```

Comment:
@morkapronczay
Anything you have to add here to set up what Superlinked Spaces will add to user acquisition analytics is welcome.

Reply:
- what are some commonalities in our user signup data that we can mine to make better decisions later when planning our next campaign?